assets/base.css

```diff
@@ -3120,3 +3120,41 @@ details-disclosure > details {
 .rte blockquote > * {
   margin: -0.5rem 0 -0.5rem 0;
 }
+
+/* Animations */
+
+.scroll-animation__trigger {
+  transition: opacity var(--duration-long) ease-out;
+  opacity: 0;
+}
+
+.scroll-animation__trigger--active {
+  animation: var(--duration-long) ease-out slideIn forwards;
+}
+
+@keyframes slideIn {
+  from {
+    transform: translateY(10vh);
+    opacity: 0;
+  }
+  to {
+    transform: translateY(0);
+    opacity: 1;
+  }
+}
+
+/* Turn off animations if JS is turned off. */
+.no-js .scroll-animation__trigger,
+.no-js .scroll-animation__trigger--active {
+  opacity: 1;
+  animation: none;
+}
+
+/* Respect the visitor's motion preferences. */
+@media (prefers-reduced-motion) {
+  .scroll-animation__trigger,
+  .scroll-animation__trigger--active {
+    opacity: 1;
+    animation: none;
+  }
+}
```

Comment:
Nit: Keep these rules next to `.scroll-animation__trigger` rules in the file since they are so closely related.